site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
+seoTitle: 'How to use WebSockets in Chrome Extension Service Workers'
+date: 2023-07-04
+description: 'Step-by-step instructions on how to connect to a WebSocket in your Chrome Extension.'
+---
+
+This tutorial demonstrates how to connect to a WebSocket in your Chrome Extension's service worker. You can find a [working example on Github](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.websockets).
```

Comment:
Would this benefit from a prereq section? 
Including extension development, familiarity with extensions service workers and  maybe include a link to MDN's WebSockets in case some developers are not familiar?

Might be overkill... you're call 👍

Reply:
Added a link to web sockets, but a prerequisite section is too much (imo prerequisite makes sense for a tutorial, but not for a how to - I know the file name is `tut`, but it's in reality a howto).

Reply:
Agreed. It wasn't a "prerequisite" to add it :) 

Thanks for adding the link. There was an issue with the bracket. I made a suggestion